Library/Homebrew/utils/analytics.sh

```diff
@@ -0,0 +1,48 @@
+if [[ -n "$HOMEBREW_ANALYTICS" && -z "$HOMEBREW_NO_ANALYTICS" ]]
+then
+  # User UUID file. Used for Homebrew user counting. Can be deleted and
+  # recreated with no adverse effect (beyond our user counts being inflated).
+  HOMEBREW_ANALYTICS_USER_UUID_FILE="$HOME/.homebrew_analytics_user_uuid"
+  if [[ -r "$HOMEBREW_ANALYTICS_USER_UUID_FILE" ]]
+  then
+    HOMEBREW_ANALYTICS_USER_UUID="$(<"$HOMEBREW_ANALYTICS_USER_UUID_FILE")"
+  else
+    HOMEBREW_ANALYTICS_USER_UUID="$(uuidgen)"
+    echo "$HOMEBREW_ANALYTICS_USER_UUID" > "$HOMEBREW_ANALYTICS_USER_UUID_FILE"
+  fi
+  export HOMEBREW_ANALYTICS_ID="UA-75654628-1"
+  export HOMEBREW_ANALYTICS_USER_UUID
+
+  if [ -z "$HOMEBREW_ANALYTICS_DEBUG" ]
+  then
+    # Send the to-be-executed command as an "App Screen View". Anonymise the IP
+    # address (aip=1) and don't send or store any personally identifiable
+    # information.
+    # https://developers.google.com/analytics/devguides/collection/protocol/v1/devguide#screenView
+    # https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters
+    "$HOMEBREW_CURL" https://www.google-analytics.com/collect \
+      --max-time 3 \
+      --user-agent "$HOMEBREW_USER_AGENT_CURL" \
+      -d v="1" \
+      -d tid="$HOMEBREW_ANALYTICS_ID" \
+      -d cid="$HOMEBREW_ANALYTICS_USER_UUID" \
+      -d aip="1" \
+      -d an="$HOMEBREW_PRODUCT" \
+      -d av="$HOMEBREW_VERSION" \
+      -d t="screenview" \
+      -d cd="$HOMEBREW_COMMAND" \
+      --silent --output /dev/null &>/dev/null & disown
+  else
+    "$HOMEBREW_CURL" https://www.google-analytics.com/debug/collect \
+      --max-time 3 \
+      --user-agent "$HOMEBREW_USER_AGENT_CURL" \
+      -d v="1" \
+      -d tid="$HOMEBREW_ANALYTICS_ID" \
+      -d cid="$HOMEBREW_ANALYTICS_USER_UUID" \
+      -d aip="1" \
+      -d an="$HOMEBREW_PRODUCT" \
+      -d av="$HOMEBREW_VERSION" \
+      -d t="screenview" \
+      -d cd="$HOMEBREW_COMMAND"
```

Comment:
Updated with this in place. See https://github.com/Homebrew/brew/blob/7cb6c73e5cf8f493a0dce925d1ab1c999043e255/Library/Homebrew/utils/analytics.sh#L27-L34